modules/installation-aws-edge-compute-pools-examples.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones)
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones) 
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-aws-edge-compute-pools-examples_{context}"]
+= Examples of installation configuration files with edge compute pools
+
+The following examples show `install-config.yaml` files that contain an edge machine pool configuration.
+
+.Configuration that uses an edge pool with a custom instance type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-edgezone
+compute:
+- name: edge
+  platform:
+    aws:
+      type: m5.4xlarge
```

Comment:
`r5.2xlarge` is available on AZ/LZ/WLZ, maybe it's a better choice for this sample, at least `m5.4xlarge` is not available in any WLZ on `us-west-2`, @mtulio WDYT?

Reply:
@yunjiang29 that's a good point, the zone used in the example must be linked to a valid type. Thanks.
@dfitzmau could you please replace the type here?

Reply:
Thank you both. Updated!